### PR TITLE
Move list of wiki maintainers to `People`

### PR DESCRIPTION
**Pull request by Walavouchey**
had this on the to-do list for a while

updated title in languages i was confident i wouldn't run into grammar shenanigans

`people` and not `people/the_team` to follow pattern of no user group

### CODE REVIEW

File: wiki/Main_Page/de.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ osu! würde nicht ohne die Hilfe von vielen Menschen existieren, die in der Entw
 
 [Das Team](/wiki/People/The_Team): [Entwickler](/wiki/People/The_Team/Developers) • [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) • [Support Team](/wiki/People/The_Team/Support_Team) • [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) • [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) • [osu! Alumni](/wiki/People/The_Team/osu!_Alumni) • [Project Loved Team](/wiki/People/The_Team/Project_Loved_Team)
 
-[Community-Mitwirkende](/wiki/People/Community_Contributors) • [Benutzer mit einzigartigen Titeln](/wiki/People/Users_with_unique_titles) • [Turnierkomitee](/wiki/People/Tournament_Committee) • [Performance-Punkte-Komitee](/wiki/People/Performance_Points_Committee)
+[Community-Mitwirkende](/wiki/People/Community_Contributors) • [Benutzer mit einzigartigen Titeln](/wiki/People/Users_with_unique_titles) • [Turnierkomitee](/wiki/People/Tournament_Committee) • [Performance-Punkte-Komitee](/wiki/People/Performance_Points_Committee) • [Verantwortlichen des osu!-Wikis](/wiki/people/osu!_wiki_maintainers)
```
Comment (RockRoller01) [inline wiki/Main_Page/de.md:119]:
```suggestion
[Community-Mitwirkende](/wiki/People/Community_Contributors) • [Benutzer mit einzigartigen Titeln](/wiki/People/Users_with_unique_titles) • [Turnierkomitee](/wiki/People/Tournament_Committee) • [Performance-Punkte-Komitee](/wiki/People/Performance_Points_Committee) • [Betreuer des osu!-Wikis](/wiki/people/osu!_wiki_maintainers)
```

File: wiki/People/de.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 Indexseite für Artikel über Personen oder Gruppen.
 
 - [Mitwirkende der Community](Community_Contributors)
+- [Verantwortlichen des osu!-Wikis](osu!_wiki_maintainers)
```
Comment (RockRoller01) [inline wiki/People/de.md:6]:
```suggestion
- [Betreuer des osu!-Wikis](osu!_wiki_maintainers)
```

File: wiki/People/osu!_wiki_maintainers/de.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
-# Liste der Verantwortlichen des osu!-Wikis
+---
+outdated_translation: true
+outdated_since: 22188ef2a9149c3f0999652c9386734bb1215ffe
+---
+
+# Verantwortlichen des osu!-Wikis
```
Comment (RockRoller01) [inline wiki/People/osu!_wiki_maintainers/de.md:6]:
```suggestion
# Betreuer des osu!-Wikis
```

**Review by RockRoller01 — CHANGES_REQUESTED** [review]
Current TL is grammatically wrong, but that doesn't matter much as the chosen Noun is meh. The article itself uses "Betreuer", tho maybe "Instandthalter" would be better, but I leave that to the people that are more familiar with the german TL

**Comment by The-Last-Cookie** [comment]
`Betreuer` sounds fine to me.

The word `Instandhalter` was never used as far as I know.

All wiki articles use both `Betreuer` and `Verantwortlicher` for the translation of maintainer to have some word variety. Maybe `Leiter` is a good alternative too? :thinking:

Regardless of this, the changes that @RockRoller01 proposes can be merged into the branch in my opinion.

**Review by TicClick — APPROVED** [review]
tentative approval from me -- not sure how many changes you want to apply to the translations right now; I included mine, the rest (merge or not yet) is up to you

**Comment by Walavouchey** [comment]
i indeed wasn't intending on updating all of them myself, just the ones that seemed feasible to do quickly (and i only changed the titles)

**Pull request merged**